core/templates/pages/exploration-player-page/templates/information-card-modal.directive.html

```diff
@@ -108,4 +108,21 @@ <h2 ng-style="titleWrapper()" class="oppia-info-card-title"><[explorationTitle]>
     cursor: default;
     margin-right: 5px;
   }
+  .oppia-info-card-content p {
+    line-height: 1.5;
+    padding-bottom: 8px;
+  }
+  .oppia-info-card-bottom-row {
+    line-height: 1.5;
+  }
+  .flex-40 {
+    padding-bottom: 2%;
+  }
```

Comment:
Where is this used?

Reply:
![image](https://user-images.githubusercontent.com/68547101/99939149-6dc2fa80-2d8f-11eb-8d87-b21386165e54.png)
this is what it looks like with the styling.

Reply:
without it it looks like this-
![image](https://user-images.githubusercontent.com/68547101/99939214-90edaa00-2d8f-11eb-9bb5-10ae9c205259.png)
it bassically creates space between the sharing link icons and the tags

Reply:
Instead of adding more properties to flex-40 (which is a generic class that could be reused again in this directive) can we add a bootstrap class in the relevant element?